This patch is needed to fix dblatex with newer versions of TeX Live.
It was taken from https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=840189

--- a/latex/style/dbk_table.sty	Sun Oct 02 23:21:03 2016 +0200
+++ b/latex/style/dbk_table.sty	Mon Oct 10 21:31:00 2016 +0200
@@ -9,7 +9,6 @@
 \usepackage{longtable}
 \usepackage{lscape}
 \usepackage{colortbl}
-\usepackage{multirow}
 \usepackage{calc}
 \usepackage{hhline}

@@ -21,16 +20,6 @@
   \expandafter[\expandafter3\expandafter]\expandafter{%
   \multicolumn{#1}{#2}{#3}}

-% Make \@xmultirow long
-\expandafter\long\expandafter\def%
-\expandafter\@xmultirow%
-\expandafter#\expandafter1%
-\expandafter[\expandafter#\expandafter2\expandafter]%
-\expandafter#\expandafter3%
-\expandafter[\expandafter#\expandafter4\expandafter]%
-\expandafter#\expandafter5%
-\expandafter{\@xmultirow{#1}[#2]{#3}[#4]{#5}}
-
 % For the newtbl code
 \newdimen\newtblstarfactor%
 \newdimen\newtblsparewidth%
